### DISCUSSION TITLE
关于分页的小建议

### DISCUSSION THREAD

**Discussion by mcnk328** [AlistGo/alist#321] [General]
现在2.0.5版本已经使用，分页功能很不错，也感谢作者的更新，关于分页的小建议是访问根目录的时候不显示加载分页的选项，因为一般多文件存储都是在二级目录或者三级目录里(可能少数人会直接存储在根目录)，所以个人感觉根目录可以不显示分页的选项。

**Comment by xhofe** [comment]
不考虑这种逻辑 之后会加个如果当前目录文件数量不超过设置的pagesize 就不显示分页
